src/app/problems/problem-edit/java-edit-snippet/java-edit-snippet.component.ts

```diff
@@ -47,16 +47,18 @@ export class JavaEditSnippetComponent implements OnInit {
     }
 
     ngOnInit(): void {
-        if (this.questionDetails.event) {
+        // TODO: refactor => typeof this.questionDetails.event === 'number'
```

Comment:
Could not determine a good way to refactor this `if` statement. Seems like a larger issue with the model that is out of the scope of the bug-fix.

Reply:
Yes!